peep.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# Get pip version number
+PIPVER=`pip --version | awk '{print $2}'`
+ARGS=""
+
+echo "peep.sh: Using pip $PIPVER"
+
+# If we're using pip 1.5, 1.6 or 6.0, then toss on the --no-use-wheel
+# argument.
+#
+# Note: If we encounter other versions of pip here that require other
+# things, we can toss them in.
+case $PIPVER in
+    1.5*|1.6*|6.0*)
+        echo "peep.sh: Wheel-using pip detected, so passing --no-use-wheel."
+        ARGS="--no-use-wheel"
+        ;;
+    6.1*)
```

Comment:
I think given you know 6.0 works and it's covered in the previous case and you know 6.1 does not work, you can probably assume all future versions of pip won't work (for now), so you could just do:

```
6*)
```